app/services/edition_filter.rb

```diff
@@ -41,6 +44,22 @@ def allowed_sort?(sort)
     SORT_KEYS.flat_map { |item| [item, "-#{item}"] }.include?(sort)
   end
 
+  def access_limited_scope(scope)
+    scope.where(
+      "access_limit_id IS NULL" + " OR " +
+      "(
+         access_limits.limit_type = 'primary_organisation' AND
+         #{TAG_CONTAINS_QUERY % { tag: 'primary_publishing_organisation' }}
```

Comment:
This constant seems to have proved rather handy at concealing the rather grim SQL 👍